media/css/firefox/challenge-the-default/index.scss

```diff
@@ -128,7 +128,7 @@ html.is-firefox {
         @include font-firefox;
         @include font-size(40px);
         font-weight: 800;
-        word-break: break-word;
+        white-space: nowrap;
```

Comment:
Question: not sure I understand this bit (it's early here and I might just need more coffee)? Is the desired behavior that the word should run off screen, and break out of the viewport, rather than break with a hyphen?

Before:

![image](https://github.com/mozilla/bedrock/assets/400117/0e2a7d6d-8dbe-4828-9d53-386f89b7d8ac)

After:

![image](https://github.com/mozilla/bedrock/assets/400117/0d8ecaad-8b91-4971-98bd-8571cf37f33a)

Reply:
Ah, so what was happening on smaller screens (I'm using an iPhone 12 mini -- so super small) was that only the `s` was wrapping to the next line.
I just removed `white-space` rule as well as kept the older `word-break` rule omitted, and now it wraps as expected.